pydantic/networks.py

```diff
@@ -154,6 +156,10 @@ def __hash__(self) -> int:
     ),
 ]
 """A type that will accept any MariaDB DSN."""
+SQLiteDsn = Annotated[
+    FilePath,
+    StringConstraints(pattern=r'.*\.db'),
+]
```

Comment:
Does this make sense? As far as I understand, you just need a file name connect to a SQLite database, so this case doesn't mirror the other `Dsn` types we support.